style.css

```diff
@@ -0,0 +1,179 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital@0;1&display=swap');
+
+html {
+  box-sizing: border-box;
+  font-size: 10px;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  margin: 0;
+  font-size: 1.5rem;
+}
```

Comment:
- [ ] I kindly suggest you to stick to Figma design, currently your design and Figma design are different:
## Figma
![image](https://user-images.githubusercontent.com/77038610/143390039-f6c2a839-f2f9-4217-acae-05cf29dcefb2.png)

## Yours: 
![image](https://user-images.githubusercontent.com/77038610/143390155-fc35d914-d71c-4acb-a8a5-52972e79f5a1.png)

 - [ ] Please use the same **background, margins, paddings and font sizes** as specified on Figma 👍🏻 

 - [ ] I suggest you to set body{ width: 375px }, this way the width will be same as on Figma, and you can make sure that your design is exactly same as the design on Figma 👍🏻
 - [ ] Once again, please make sure that your design is exactly the same  🤗